services/QuillLMS/client/setEnvVars.js

```diff
@@ -1 +1 @@
-process.env.GOLANG_FANOUT_URL = 'https://us-central1-comprehension-247816.cloudfunctions.net/comprehension-fanout-staging';
+process.env.GOLANG_FANOUT_URL = 'https://staging.comprehension-feedback.quill.org/';
```

Comment:
This change is not directly related to the feature, but is required in order to run the feature locally and should be in place for everyone.